include/gatekeeper_net.h

```diff
@@ -76,6 +76,8 @@ int lua_init_iface(struct gatekeeper_if *iface, const char *iface_name,
 void lua_free_iface(struct gatekeeper_if *iface);
 
 struct net_config *get_net_conf(void);
+struct gatekeeper_if *get_if_front(void);
+struct gatekeeper_if *get_if_back(void);
```

Comment:
`get_if_front()` and `get_if_back()` should take `struct net_config` as their single parameters. Although there's a single instance of `struct net_config` inside of Gatekeeper, we should not require this property throughout the code because if we ever decide to remove this property, there'll be a lot of extra work. Not to mention that this singleton property does not define Gatekeeper, so the possibility of change is a real one.